build/Glyssen.proj

```diff
@@ -199,23 +199,47 @@
 		<Delete Files="$(RootDir)\output\RefTextUtilityInstallerinstaller\RefTextUtilityInstaller.msi" TreatErrorsAsWarnings="false" />
 
 	</Target>
+	
+	<!-- See https://wixtoolset.org/documentation/manual/v3/overview/insignia.html -->
+	<Target Name="SignCabs">
```

Comment:
Does it work having the targets in this file? From the documentation I'd have thought that they should be in `Installer.wixproj`.

Reply:
I think you're right. It's been over a week now and I can only just barely remember what I was doing, but that's probably why I marked it WIP.